Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/classic/MetaUICustomizationClassicUnitFragment.kt

```
@@ -0,0 +1,158 @@
+                }
+
+                override fun onAdReceiveSuccess() {
+                    super.onAdReceiveSuccess()
+                    Log.d(TAG, "onAdReceiveSuccess")
+                }
+
+                override fun onAdReceiveFail(error: String?) {
+                    super.onAdReceiveFail(error)
+                    Log.d(TAG, "onAdReceiveFail $error")
+                }
+
+                override fun onResize(height: Int) {
+                    super.onResize(height)
+                    Log.d(TAG, "onResize $height")
+                }
+            })
+        tblMetaClassicUnit.setMetaAdTypeForDebug(MetaConst.TEST_LAYOUT_IMAGE_LINK_TYPE)
+        tblMetaClassicUnit.setUnitExtraProperties(hashMapOf(MetaConst.AUDIENCE_NETWORK_PLACEMENT_ID_KEY to MetaConst.AUDIENCE_NETWORK_PLACEMENT_ID))
+
+        val typefaceArielBold = Typeface.Builder(
+            requireActivity().assets,
+            getFullPathToFontFile(FONT_TYPEFACE_ARIAL_BOLD)
+
+        ).build()
+
+        // Create custom style properties for the title
+        val titleStyleProperties = TBLTitleStylePropertiesBuilder()
+            .setAmountOfSpaceBetweenLines(AMOUNT_OF_SPACE_BETWEEN_ELEMENTS)
+            .setLines(NUMBER_OF_LINES)
+            .setFontSize(TEXT_FONT_SIZE)
+            .setFontLightColor(Color.Blue.toArgb())
+            .setFontDarkColor(Color.Red.toArgb())
+            .setTypeface(typefaceArielBold)
+            .build()
+
+        // Create custom style properties for the branding
+        val brandingStyleProperties =
+            TBLTextStylePropertiesBuilder(ELEMENT_TYPE_BRANDING)
+                .setFontSize(TEXT_FONT_SIZE)
+                .setFontLightColor(R.color.purple_200)
+                .setFontDarkColor(R.color.purple_500)
+                .setTypeface(typefaceArielBold)
+                .build()
+
+//        // Create custom style properties for the call to action button
+//        val callToActionButtonStyleProperties =
```

@DanielFrTB - Why is this code commented out? As mentioned in my previous comment, show the option to publisher that is NOT the default (meaning, hide the CTA).
Leaving commented out code is not a good practice.

---

@tomer-br-taboola 
Mosh ask to put here a commented code for demonstration